fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneRecordCursor.java

```diff
@@ -232,14 +241,18 @@ private synchronized IndexReader getIndexReader() throws IOException {
     private void performScan() throws IOException {
         indexReader = getIndexReader();
         searcher = new IndexSearcher(indexReader);
+        int limit = min(limitRemaining, 200);
```

Comment:
Make `200` a static field. It's fine for now that it's constant, but at least then there's a place for some Javadoc.